Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -94,10 +94,10 @@ uplink-agent-id-card-name = Agent ID Card
 uplink-agent-id-card-desc = A modified ID card that can copy accesses from other cards and change its name and job title at-will.
 
 uplink-black-jetpack-name = Black Jetpack
-uplink-black-jetpack-desc = A black jetpack. It allows you to fly around in space. Additional fuel now included!
+uplink-black-jetpack-desc = A black jetpack. It allows you to fly around in space. Additional fuel not included!
```

Comment:
im pretty sure they come filled

Reply:
They do, but it says "Additional" fuel. Which I interpreted to mean "for when it runs out".
 I was a bit puzzled why this sentence is there in the first place. Might be best to just snip the extra sentence